### ISSUE TITLE
Suppress preview card for posts with media

### ISSUE THREAD

**Issue by prohr** [elk-zone/elk#937]
Preview cards are a gorgeous alternative to just a barren link. They can really spiff up a post + make it visually interesting. Publishers love them because it increases the odds you'll click through to their site.

However, when the author has taken the time + effort to choose media to display alongside their words, preview cards should NOT be displayed.  Three examples:

1. Thread discussing two images from one linked site. Injecting the preview card is a distraction. https://elk.zone/hachyderm.io/@pevohr/109664873250118606

2. Artbot linking to works by a single artist. Injecting a preview card makes it look like the author was stuttering. https://elk.zone/mastodon.arthaus.social/@AugustMacke/109662774198259109

3. Post celebrating a bit of whimsy found online. Injecting the preview card turns it into an advertisement. https://elk.zone/norden.social/@czottmann/109663886482523238

In all three cases, the author built a post that works better without the preview card. Please honor that, because we want this to be an author-centric medium (where people discuss + share things they find interesting), not a publisher-centric clicky-clicky link farm.

SUGGESTED FIX

Only add preview cards for a link if there's zero attached media.

**Comment by mediapinta** [comment]
> It's even worse for my 3rd example:
> 
> ![image](https://user-images.githubusercontent.com/13510546/211812161-d15cc1cb-1985-403e-98b7-3f81a2079c43.png)
> 
> Technically it's "just a preview", but it looks like somebody from corporate injected an ad, which totally ruins Carlo's gentle, whimsical fun here. Nobody wants that.

Yes, I want that. Because I don't know what calm.com is and preview card saved me a clic.

btw, I like this discussion. I agree that sometimes preview cards can show redundant information and they aren't the most aesthetic, but, in the other hand, sometimes they give you context and useful information.

Looking forward how this ends.
